netstandard/ref/System.cs

```diff
@@ -2751,6 +2751,113 @@ public partial class MemberAccessException : System.SystemException
         public MemberAccessException(string message) { }
         public MemberAccessException(string message, System.Exception inner) { }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct Memory<T>
```

Comment:
Why it does not implement IEquatable<T> ?

Reply:
Equality for memories is hard; we talked about this a lot and concluded that it's not something we feel warm and fuzzy about. @GrabYourPitchforks or @jkotas might remember.

However, this isn't the place the to discuss the API design. We can only decide to not expose APIs, we shouldn't redesign them here.

Reply:
Well, if it's hard why do you have `public bool Equals(System.Memory<T> other)` ? I think it'd make sense to have the interface here if the implementation is there too.